src/util/feefrac.h

```diff
@@ -182,6 +182,27 @@ struct FeeFrac
         std::swap(a.fee, b.fee);
         std::swap(a.size, b.size);
     }
+
+    /** Compute, at this object's feerate, how much fee does at_size correspond to.
+     *
+     * This effectively corresponds to evaluating (this->fee * at_size) / this->size, with the
+     * result rounded down (even for negative feerates).
+     *
+     * Requires this->size > 0, at_size >= 0, and that the correct result fits in a int64_t. This
+     * is guaranteed to be the case when 0 <= at_size <= this->size.
+     */
+    int64_t Evaluate(int32_t at_size) const noexcept
+    {
+        Assume(size > 0);
+        Assume(at_size >= 0);
+        if (fee >= 0 && fee < 0x200000000) [[likely]] {
```

Comment:
would it make sense to add non-fuzz `0x1ffffffff`, `0x200000000` and `0x200000001` test cases for this boundary?